prisma/schema.prisma

```diff
@@ -204,11 +202,29 @@ model Judgement {
 }
 
 model ReactionCount {
-  type ReactionType
-  count Int @default(0)
+  type  ReactionType
+  count Int          @default(0)
 
   project   Project @relation(fields: [projectId], references: [id])
   projectId String
 
   @@id([type, projectId])
 }
+
+model Photo {
+  id        String   @id @default(cuid())
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  url       String
+  urlLarge  String
+  urlMedium String
+  urlSmall  String
+
+  thanks String?
+
+  programId    String? @default("codeday")
```

Comment:
eventId and programId _shouldn't_ be nullable, and eventGroupId and regionId _should_ be nullable, that way it's the same as the model for Project.